products/amq/get-started.adoc

```diff
@@ -20,11 +20,11 @@ Prerequisites
 == Step1 Content
 
 1. *Install JBoss Developer Studio*
-a. Download link:http://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-standalone_jar.jar?tp=amq[JBoss Developer Studio 8.1.0.GA] and execute its installer.
-* On Mac / Windows development hosts: Right click on `jboss-devstudio-8.1.0.GA-installer-standalone.jar`, select _Open With -> Jar Launcher_
+a. Download link:https://developers.redhat.com/download-manager/content/origin/files/sha256/1e/1efef4d2b109efd9e8136d02067a8d62e00621e5b2b1530811bf52b5aa2fca6f/jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar?tp=amq[JBoss Developer Studio 9.0.0.GA] and execute its installer.
+* On Mac / Windows development hosts: Right click on `jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar`, select _Open With -> Jar Launcher_
 * On Red Hat Enterprise Linux: Use the terminal to go to the folder where you have downloaded JBoss Developer Studio, and execute the installer with the following command:
 +
-`java -jar jboss-devstudio-8.1.0.GA-installer-standalone.jar`
+`java -jar jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar`
```

Comment:
This should be changed the 8.1.0 eap bundle jar, not the 9.0 one.

https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-jar_universal.jar